Cargo.toml

```diff
@@ -35,6 +35,9 @@ tailwind_fuse = { version = "0.3", features = ["variant"] }
 web-sys = { version = "0.3", optional = true, features = ["Url"] }
 url = { version = "2.5", optional = true }
 
+[dev-dependencies]
+anyhow = "1.0.86"
```

Comment:
I don't see this being used anywhere in the code. I don't think we need it at the moment.

Reply:
I had used it in a test, but I'll remove it now.